project/Build.scala

```diff
@@ -1452,6 +1438,7 @@ object Build {
           name := "JavaLib Ex Test Suite",
           publishArtifact in Compile := false,
           testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a", "-s"),
+          scalacOptions += "-Ynowarn-default-junit-methods",
```

Comment:
I had to add the `-Ynowarn-default-junit-methods` explicitly `javalibExTestSuite` because for this project the plugin fails to set the flag. The flag is set using a `tryToSet` in `global.settings.nowarnDefaultJunitMethods.tryToSet(List("true"))` so it is not so surprising that it failed.